### PR TITLE
Epping Forest District Council

### LINKED ISSUE #554: Please add Epping Forest to Council List
### Name of Council

Epping Forest

### Example Address/Postcode

IG9 6EP

### Additional Information

https://www.eppingforestdc.gov.uk/recycling-and-rubbish/check-your-collection/

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [X] I have provided a tested working address/postcode/UPRN with bin collections available
- [X] I understand that this project is run by volunteer contributors and completion depends on numerous factors - even with a request, we cannot guarantee if/when your council will get a script

### PR DESCRIPTION
**Pull request by raddzz**
Adding support for Epping Forest District Council should close #554

### CODE REVIEW

File: uk_bin_collection/tests/features/validate_council_outputs.feature
Comment (dp247) [inline uk_bin_collection/tests/features/validate_council_outputs.feature:1]: Lines 200-204 need their indentation moving back 😉

**Review by dp247 — CHANGES_REQUESTED** [review]
Looks good - just need the indentation fixing on the feature file

File: uk_bin_collection/tests/features/validate_council_outputs.feature
Comment (robbrad) [reply uk_bin_collection/tests/features/validate_council_outputs.feature:1]: @raddzz ^

**Comment by robbrad** [comment]
added to PR https://github.com/robbrad/UKBinCollectionData/pull/739

**Pull request closed without merging**